Scheduler/observing_blocks_maintel/AOS-OR4/BLOCK-282.json

```diff
@@ -0,0 +1,202 @@
+{
+    "name": "BLOCK-282",
+    "program": "BLOCK-282",
+    "constraints": [],
+    "scripts": [
+        {
+            "name": "run_command.py",
+            "standard": true,
+            "parameters": {
+                "component": "CCCamera",
+                "cmd": "play",
+                "parameters": {
+                    "playlist": "or4_day2_aos_sweep_cam_rx",
+                    "repeat": true
+                }
+            }
+        },
+        {
+            "name": "maintel/slew_to_azel.py",
+            "standard": true,
+            "parameters": {
+                "track_azel": {
+                    "az": 9.0,
+                    "el": 75.0
+                },
+                "rot_value": -45.0,
```

Comment:
Looks like this was intended to be +45.0: https://github.com/jmeyers314/aos_block_planner/blob/main/OR4_AOS_schedule.ecsv#L208